### DISCUSSION TITLE
support for this type of adapter

### DISCUSSION THREAD

**Discussion by Didazne** [chris1111/Wireless-USB-Big-Sur-Adapter#263] [General]
![PXL_20230319_150904331](https://user-images.githubusercontent.com/128307748/226185808-cc5e76fc-94c7-4b7f-978f-fc6a9e28a461.jpg)
@chris1111 
Does the driver work for this type of adapter?

**Comment by chris1111** [comment]
I do not see any model so test them and you will see

**Reply by nonita183** [reply]
hi Chris . I am desperately in need of help for making my Tp link wifi dongle work on my mac OS Monterey. I followed the steps you shared on this link community.tp-link.com/en/home/forum/topic/239134. i downloaded the driver but it is not working. the error is "The developer of this app needs to update it to work with this version of macOS.Contact the developer for more information." I request you to please please help me, i was in the middle of an assignment when i updated my mac book and alas, my internet is gone!!!

**Reply by chris1111** [reply]
> hi Chris . I am desperately in need of help for making my Tp link wifi dongle work on my mac OS Monterey. I followed the steps you shared on this link community.tp-link.com/en/home/forum/topic/239134. i downloaded the driver but it is not working. the error is "The developer of this app needs to update it to work with this version of macOS.Contact the developer for more information." I request you to please please help me, i was in the middle of an assignment when i updated my mac book and alas, my internet is gone!!!



Follow [instruction here, ➥ Wireless-USB-Big-Sur-Adapter](https://github.com/chris1111/Wireless-USB-Big-Sur-Adapter) not from other link
